app/models/application_record.rb

```diff
@@ -0,0 +1,5 @@
+require 'active_record'
```

Comment:
@noremmie , I saw your comment on this. You are correct that that line shouldn't be necessary. I am able to run `rails c` without issue in my env on this branch, without the `require`. LMK if you want to look at the error together.